moneybot/markets/history.py

```
@@ -54,23 +60,24 @@ def contemporary_usd_price(row) -> float:
 
     def historical_prices_of(
         pair: str,
-        period=900,
-        start=YEAR_AGO,
-        end=time.time(),
-    ) -> pd.Series:
+        period: int = 900,
+        start: Optional[float] = None,
+        end: Optional[float] = None,
+    ) -> Series:
         '''
         Returns a series of time-indexed prices.
 
         `pair` is of the form e.g. 'BTC_ETH',
         `period` is an integer number of seconds,
         either 300, 900, 1800, 7200, 14400, or 86400.
         '''
-        ex_trades = polo.returnChartData(pair,
-                                         period,
-                                         start, end)
-        ts_df = pd.DataFrame(ex_trades, dtype=float)
-        ts_df.index = [datetime.fromtimestamp(t)
-                       for t in ts_df['date']]
+        now = time.time()
+        start = start or now - YEAR_IN_SECS
```

Yeah I think that in that case it should probably just throw an error and say "go get some data, dummy."